### ISSUE TITLE
Upgrade to new version of Stackable or deprecate plugin

### ISSUE THREAD

**Issue by cjyabraham** [LF-Engineering/lfevents#418]
[Version 2 is out](https://wpstackable.com/blog/version-2-is-out/).  Test first to see if it breaks any blocks and layouts.  It contains responsive image loading.

**Comment by cjyabraham** [comment]
Some experimenting with this did not go well.  We will need to deploy this following these steps:

1. Install on branch; activate plugin
2. Go through much of the site content and save each page to put each block on v2; identify broken layouts; some already identified:
- `section-header`s don't reach the edge of the screen
- feature grid boxes [here](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/attend/about-shanghai/) get left-justified against edge of browser and then, when re-saved, the images go narrow
3. Update theme css to account for different stackable styling that breaks site
4. Deploy and re-save every Event page to put each block on v2; Identify and fix any lingering design issues
5. Turn off deprecated css/js that stackable v2 initially installs with.  (Stackable v2 runs with 2 extra css/js files for deprecated styling.  By re-saving every Event on the site, I suspect this will put us on all the v2 versions of the blocks so we don't need to carry these extra files going forward.  This needs testing)

This is a huge task and will need @andycochran and I working in parallel to accomplish this.  I suspect it will take maybe 2 full days of @andycochran and I working together on this to do the upgrade.

We should assess the importance of doing this and staying up-to-date with stackable.  More [info on v2 is here](https://wpstackable.com/blog/version-2-is-out/).  On the one hand, in the future with changing browser technology, staying on the latest version of stackable could be important, now that we've committed to it with our content.  It may also be clearer in the future, as v2 matures, that there are new features we really want and will end up justifying this upgrade.  For example, apparently v2 has responsive image loading for different device types.

Alternatively, we may decide to phase out Stackable blocks as the native Gutenberg blocks mature and make Stackable no longer needed for what we are doing.  In which case, we'd just need to keep the existing version around for old Events but there'll be no need to upgrade.  It might be good to try setting up an Event using only Gutenberg blocks and seeing if we can get them to work @meganelizabethphillips?  I would also like to assess the increase in weight of the js/css files for v2 and whether these are really moving us forward technologically or just weighing us down with features we don't need.  To be continued...

**Comment by cjyabraham** [comment]
Here's a [fake kubecon page](https://sandbox-lfeventsci.pantheonsite.io/fake-kubecon/) using a Gutenberg Cover block instead of a Stackable Container

**Comment by cjyabraham** [comment]
Old version of stackable css/js: 182KB
New version of stackable with deprecated css/js: 364KB
New version of stackable without deprecated css/js: 178KB

**Comment by cjyabraham** [comment]
I spoke with @andycochran today and he agrees that we should be moving off of stackable now that Gutenberg blocks have matured.  @meganelizabethphillips, let's discuss next time we talk.

**Comment by cjyabraham** [comment]
Here's a quick summary of how Stackable blocks are currently used on the site:
* Hero Container block; easily replaced with a Cover block. [Example](https://sandbox-lfeventsci.pantheonsite.io/fake-kubecon/).
* Section Header; can be replaced with a Cover block but can’t configure gradient; could we use just a solid color for this?  Group header also works.  [Examples](https://sandbox-lfeventsci.pantheonsite.io/fake-kubecon/).  [Gutenberg issue tracking custom gradients](https://github.com/WordPress/gutenberg/issues/16662).  We could either wait for Gutenberg to implement configurable gradients or find a way to override the colors with a class applied to the block.
* Schedule-at-a-glance Container block; can be replaced by Cover or Group but can’t currently configure gradient.
* Feature grid on [China site](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/attend/about-shanghai/)
* Image box on [this page](https://dev-lfeventsci.pantheonsite.io/lf-member-summit/attend/about-lake-tahoe/) and [here](https://dev-lfeventsci.pantheonsite.io/agl-amm/attend/about-waikiki/) and [here](https://dev-lfeventsci.pantheonsite.io/open-networking-edge-summit-north-america/attend/about-los-angeles/) and [here](https://dev-lfeventsci.pantheonsite.io/xen-summit/attend/about-bucharest/)

**Comment by cjyabraham** [comment]
Closing in favor of #436
